contrib/run_task.sh

```diff
@@ -242,6 +244,13 @@ need_cmd() {
     fi
 }
 
+need_nightly() {
+    cargo_ver=$(cargo --version)
+    if echo "$cargo_ver" | grep nighly; then
```

Comment:
```suggestion
    if echo "$cargo_ver" | grep nightly; then
```

Reply:
Also, shouldn't this be `grep -v`?

Reply:
Yes :) two mistakes cancelled themselves out.

Reply:
`shellcheck` would have caught that I rekon.

Reply:
`shellcheck` isn't able to check spelling nor can it tell whether I meant `-v` ... but I think ChatGPT would've caught this had I run the diff through there.

Reply:
Oh of course, i was thinking nighly was an unset variable - face palm. 

> but I think ChatGPT would've caught this had I run the diff through there.

So you just paste a diff and ask "check this" and it gives you useful feedback? What version?